go.mod

```diff
@@ -123,7 +133,9 @@ require (
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/gorilla/handlers v1.5.2 // indirect
 	github.com/gorilla/websocket v1.5.0 // indirect
-	github.com/hashicorp/errwrap v1.0.0 // indirect
+	github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c // indirect
+	github.com/hamba/avro/v2 v2.22.2-0.20240625062549-66aad10411d9 // indirect
```

Comment:
This can not be introduced, [CVE-2023-37475](https://www.cve.org/CVERecord?id=CVE-2023-37475), already been tagged by Intuit security.